### DISCUSSION TITLE
AGH taking down ethernet adapter?

### DISCUSSION THREAD

**Discussion by thedinz** [AdguardTeam/AdGuardHome#2737] [Q&A]
I having a weird issue where after running AGH for a few hours on docker the machine running docker's ethernet adapter will show as if nothing is connected to it. Very odd behavior, I have tried everything but the only fix is to restart the docker container for ad guard and then the ethernet adapter detects it's plugged in.

Now keep in mind it's not just the DNS is not resolving, it actually thinks there is no ethernet cable plugged in. Very odd. The machine running AGH doesn't even route through it I have manually specified google DNS so that any issues with AGH and the machine should still be accessible at least when I was using pihole it worked this way.

What could AGH be doing after a few hour uptime that makes the server suddenly think there is no ethernet cable plugged into it?

**Comment by ameshkov** [comment]
Nothing really, AGH just listens to the specified address and that's all.

**Comment by thedinz** [comment]
so the machine that runs AGH has its own DNS hardcoded in the event that if AGH took a nose dive i could still get in and when i run pihole that failover works fine. but not with AGH, when it takes a nose dive (and not sure why thats happening either) it take my whole network down like i said all machines internet icon goes disconnected, which is not the case with pihole with pihole everything appears fine but things cant talk because the DNS is not responding.... make sense.

What doesn't make sense is whats happening when AGH goes down, and you got no idea why?

Id love to move from pihole to AGH but its too unstable, it goes down once per day (no idea why) and i cant fix it unless im on site.... Thats not good.

**Comment by ameshkov** [comment]
> What doesn't make sense is whats happening when AGH goes down, and you got no idea why?

Well, it's hard to say without seeing the logs.

1. [Configure](https://github.com/AdguardTeam/AdGuardHome/wiki/Configuration#configuration-file) AdGuard Home to collect logs:
    * Specify `log_file`
    * Set `verbose` to `True`
2. Restart AdGuard Home and reproduce the issue
3. Post the log file here.

**Comment by thedinz** [comment]
[log.txt](https://github.com/AdguardTeam/AdGuardHome/files/5874055/log.txt)

Here you are, it crapped out at some point early this morning, hopefully, it easy to spot

**Comment by ameshkov** [comment]
> Here you are, it crapped out at some point early this morning, hopefully, it easy to spot

Do you remember when exactly? I see that it was restarted a couple of times.
